resources/views/print/modify.blade.php

```diff
@@ -1,15 +1,6 @@
 @if (Gate::allows('print.modify'))
     <div class="card">
         <div class="card-header">@lang('print.modify')</div>
-        @if ($errors->any())
-            <div class="alert alert-danger">
-                <ul>
-                    @foreach ($errors->all() as $error)
-                        <li>{{ $error }}</li>
-                    @endforeach
-                </ul>
-            </div>
-        @endif
```

Comment:
Shouldn't it be in an other patch? How is this change serve this change?

Reply:
Moved it to it's place, it's relevant since my function needs too.